src/main/java/hw_6/Task1.java

```diff
@@ -0,0 +1,42 @@
+package hw_6;
+
+public class Task1 {
+    public static void main(String[] args) {
+
+        Task1 task = new Task1();
+        task.NonStaticMethod();
+
+        StaticMethod();
+
+    }
+
+    int x = 3;
+    int y = 4;
+
+    static int x1 = 5;
+    static int y1 = 8;
+
+    private void NonStaticMethod() {
```

Comment:
method name should begin from the small letter

Reply:
fixed